<?xml version="1.0"?>
<!--
Launch the path follower with the input scaling controller for a bi-steerable robot.
-->
<launch>
    <group ns="path_follower">
        <group ns="controller">
            <!-- parameters, that are dependent of the controller or the robot model -->
            <group ns="2steer_inputscaling">
                <param name="vehicle_length" value="0.34" />
                <param name="k_forward" value="4.5" />
                <param name="k_backward" value="7.0" />
                <param name="max_steering_angle" value="0.52359877559" /> <!-- 30Â° -->
                <param name="max_steering_angle_speed" value="4.0" /> <!-- ~4.0 should be realistic -->
            </group>
        </group>
    </group>
</launch>
